nixos/modules/services/web-apps/healthchecks.nix

```diff
@@ -0,0 +1,185 @@
+{ config, lib, pkgs, buildEnv, ... }:
+
+with lib;
+
+let
+  cfg = config.services.healthchecks;
+  pkg = pkgs.healthchecks;
+  boolToPython = b: if b then "True" else "False";
+  environment = {
+    PYTHONPATH = pkg.pythonPath;
+    STATIC_ROOT = cfg.dataDir + "/static";
+    SECRET_KEY_FILE = cfg.secretKeyFile;
+    DB_NAME = "${cfg.dataDir}/healthchecks.sqlite";
+    ALLOWED_HOSTS = lib.concatStringsSep "," cfg.allowedHosts;
+    DEBUG = boolToPython cfg.debug;
+    REGISTRATION_OPEN = boolToPython cfg.registrationOpen;
+  } // cfg.settings;
+
+  environmentFile = pkgs.writeText "healthchecks-environment" (lib.generators.toKeyValue {} environment);
+
+  healthchecksManageScript = with pkgs; (writeShellScriptBin "healthchecks-manage" ''
+    sudo -u healthchecks sh -s "$@" <<"EOF"
+      export $(cat ${environmentFile} | xargs);
+      ${pkg}/bin/healthchecks-manage "$@"
+    EOF
+  '');
+in
+{
+  options.services.healthchecks = {
+    enable = mkEnableOption "healthchecks";
```

Comment:
This would be a good spot to include the expectation that generally this software is to be run behind a reverse proxy. I didn't see any instructions in upstream documentation so presumably this is a trivial task and simply mentioning a reverse proxy is sufficient.

Reply:
I've added a short sentence, is that good?

Reply:
LGTM :+1: